src/applications/ivc-champva/10-7959C/chapters/medicareInformation.js

```diff
@@ -75,6 +82,54 @@ export const applicantMedicarePartACarrierSchema = {
   }),
 };
 
+export const appMedicareOver65IneligibleUploadSchema = {
+  uiSchema: {
+    applicants: {
+      'ui:options': {
+        viewField: ApplicantField,
+      },
+      items: {
+        ...titleUI(
+          ({ _formData, formContext }) =>
+            `Upload proof of Medicare ineligibility ${isRequiredFile(
+              formContext,
+              requiredFiles,
+            )}`,
+          ({ formData }) => {
+            const appName = applicantWording(formData, undefined, false);
+            return (
+              <>
+                <b>{appName}</b> is 65 years or older and you selected that{' '}
+                they’re not eligible for Medicare.
+                <br />
+                <br />
+                You’ll need to submit a copy of a letter from the Social
+                Security Administration that confirms that <b>{appName}</b>{' '}
+                doesn’t qualify for Medicare benefits under anyone’s Social
+                Security number.
+                <br />
+                If you don’t have a copy to upload now, you can send one by mail
+                or fax
+              </>
+            );
+          },
+        ),
+        ...fileUploadBlurb,
+        applicantMedicareIneligibleProof: fileUploadUI({
+          label: 'Upload proof of Medicare ineligibility',
+        }),
+      },
+    },
+  },
+  schema: applicantListSchema([], {
+    titleSchema,
+    'view:fileUploadBlurb': blankSchema,
+    applicantMedicareIneligibleProof: fileWithMetadataSchema([
+      'Letter from the SSA',
+    ]),
+  }),
+};
+
```

Comment:
This lays out the custom markup and fileupload input for the medicare ineligibility screen